### ISSUE TITLE
[International Draughts] Result recognition doesn't works

### ISSUE THREAD

**Issue by IgorLeMasson** [mi-g/joclyboard#10]
After a game finish, the result is not showed to the user and the PDN **[Result](url=http://pdn.fmjd.org/pdntags.html#mandatory-tags)** tag is not written in PDN-like (.pjn) game notation file.

**Comment by IgorLeMasson** [comment]
`[JoclyGame "draughts"]
[Date "2017.6.4"]
[White "Player A"]
[Black "Player B"]
[PlyCount "135"]

1. 31-26 17-21 2. 26x17 11x22 3. 32-28 18-23 4. 28x17 12x21 5. 37-32 21-27 6. 32x21 16x27 7. 41-37 7-11 8. 37-32 11-16 9. 32x21 16x27 10. 33-29 6-11 11. 29x18 13x22 12. 42-37 11-16 13. 37-31 19-23 14. 39-33 22-28 15. 33x22 27x18 16. 31-27 8-13 17. 34-29 23x34 18. 40x29 18-23 19. 29x18 13x31 20. 36x27 9-13 21. 38-32 13-19 22. 44-39 4-9 23. 39-33 2-8 24. 33-28 19-23 25. 28x19 14x23 26. 43-38 9-14 27. 45-40 1-7 28. 38-33 23-29 29. 33x24 20x29 30. 40-34 29x40 31. 35x44 14-19 32. 44-39 3-9 33. 46-41 8-12 34. 41-36 9-13 35. 48-42 10-14 36. 42-38 13-18 37. 36-31 18-22 38. 27x18 12x23 39. 39-33 7-12 40. 31-27 5-10 41. 27-21 16x27 42. 32x21 14-20 43. 38-32 20-24 44. 33-28 12-18 45. 21-16 15-20 46. 16-11 23-29 47. 11-7 18-23 48. 7-1 10-14 49. 50-45 24-30 50. 28-22 30-34 51. 22-18 23x12 52. 1x18 19-23 53. 18-7 20-24 54. 47-42 14-19 55. 49-44 34-39 56. 44x33 29x47 57. 7x15 19-24 58. 15x29 47x20 59. 32-28 20-24 60. 28-22 24-35 61. 22-17 35-8 62. 17-11 8-12 63. 11-6 12-1 64. 45-40 1x45 65. 6-1 45-50 66. 1-6 50-45 67. 6-1 45-7 68. 1x12
`

**Comment by mi-g** [comment]
I agree the result does not show in the PDN, but doesn't the result appear in the board window footer, like _Player A wins_ ?

**Comment by IgorLeMasson** [comment]
The result is displayed only for certain end positions. It is not so easy to see it. 
Could you enhance the visibility, changing the text format to Bold ?!

There are positions such as 1 king vs 1 king, for which the draw result is not recognized. 
I guess that "basic knowledge" for  endgame database of 2 and 3 pieces isn't yet supported. Am I right?

**Comment by mi-g** [comment]
>  Could you enhance the visibility, changing the text format to Bold ?!

Good idea.

> There are positions such as 1 king vs 1 king, for which the draw result is not recognized.
> I guess that "basic knowledge" of 2 and 3 pieces endgame database isn't yet supported. Am I right?

It's possible, this would be a bug. Can you share a saved game where the end is not recognized ?

**Comment by IgorLeMasson** [comment]
At **0.9.12** version, the [PDN result](http://pdn.fmjd.org/pdntags.html#result-tag) tag is not written in PJN (PDN-like) file.
The result appears on Board window when a game is finished but it is not properly and explicitly shown. 

In addition, if you click on History window and back to the Board window, the result description disappears and the PDN result tag is not available on History window.

![result_tag_screen1](https://user-images.githubusercontent.com/8975474/27524918-d175320c-5a0f-11e7-9f21-6197de55c4d0.png)

Regarding the second reported issue about **endgame database** support, I guess it should be a subject of another issue, but you can see that 1 king vs 1 king was not recognized as draw at move 64 of this game: [game2_international_draughts.zip](https://github.com/mi-g/joclyboard/files/1101053/game2_international_draughts.zip)
